tests/playwright.config.ts

```diff
@@ -0,0 +1,113 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+import { devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  testDir: './integration',
+  /* Maximum time one test can run for. */
+  timeout: 30 * 1000,
+  globalTimeout: 2 * (60 * 1000), // 2 minutes
+  expect: {
+    /**
+     * Maximum time expect() should wait for the condition to be met.
+     * For example in `await expect(locator).toHaveText();`
+     */
+    timeout: 5000
+  },
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  workers: 3,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: [
+    ['html', { open: 'never', outputFolder: 'results/playwright-report' }],
+    ['list', { printSteps: true }]
+  ],
+  /* Global setup to preserve auth */
+  globalSetup: require.resolve('./global-setup'),
```

Comment:
allows auth to be saved and used by all the tests instead of each test having to login